docs/docs/web-components.md

```diff
@@ -0,0 +1,59 @@
+---
+id: web-components
+title: Web Components
+permalink: docs/web-components.html
+---
+
+React and [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) are built to solve different problems.  Web Components provide strong encapsulation for reusable components, while React provides a declarative library that keeps the DOM in sync with your data. The two goals are complementary. As a developer, you are free to use React in your Web Components, or to use WebComponents in React, or both.
+
+Most people who use React don't use Web Components at all. Unless you are using or creating third-party UI components, Web Components are likely not useful for building an app.
```

Comment:
FWIW @spicyj specifically requested adding this exact paragraph 😛 Well, I rewrote it to be more polite towards web components, I believe everyone will be happy now